package config;

message TransferSvrConfig {
	optional string listen_ip = 1;
	optional int32 svr_inst_id = 2;
	optional int32 server_zone = 3;
}